Fix:

error: assigning to 'char *' from 'unsigned char *' converts between pointers to integer types where one is of the unique plain 'char' type and the other is not

https://sourceforge.net/p/agg/svn/142/
--- font_freetype/agg_font_freetype.cpp.orig	2006-10-08 23:06:54.000000000 -0500
+++ font_freetype/agg_font_freetype.cpp	2024-09-21 07:18:08.000000000 -0500
@@ -186,7 +186,7 @@
             v_control = v_start;
 
             point = outline.points + first;
-            tags  = outline.tags  + first;
+            tags  = (char*)(outline.tags)  + first;
             tag   = FT_CURVE_TAG(tags[0]);
 
             // A contour cannot start with a cubic control point!
